### PR TITLE
Add example UpSet plot

### PR DESCRIPTION
**Pull request by jeffbaumes**
Adding a draft of the UpSet plot to the search results view. It currently uses dummy data.

image

### CODE REVIEW

**Comment by subdavis** [comment]
Oh wow, neat.

**Comment by kfagnan** [comment]
Cool!

File: web/src/views/Search/Search.vue [hunk 1]
```diff
@@ -89,6 +113,15 @@ export default Vue.extend({
     <sidebar />
     <v-main v-if="typeResults">
       <v-container fluid>
+        <v-row>
+          <v-col :cols="12">
+            <ChartContainer>
```
Comment (jeffbaumes) [inline web/src/views/Search/Search.vue:118]: @subdavis I followed the pattern in another visualization (DateHistogram/Histogram2) here for ChartContainer and how to listen for resizes in the UpSet component. Am I using it as intended here? It's a very nice piece of machinery for helping update vis on resize and it seems to work.

Comment (subdavis) [reply web/src/views/Search/Search.vue:118]: Looks good to me!

**Pull request merged**